### PR TITLE
pages.pt_BR/freebsd: add Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by rafaelgpaves**
Adding translation for the remaining `freebsd` pages.

- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.pt_BR/freebsd/base64.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Codifica um arquivo para `stdout`:
+
+`base64 {{-i|--input}} {{caminho/para/arquivo}}`
+
+- Codifica um arquivo para o arquivo de saída especificado:
+
+`base64 {{-i|--input}} {{caminho/para/arquivo_de_entrada}} {{-o|--output}} {{caminho/para/arquivo_de_saída}}`
+
+- Enrola saída codificada a uma largura específica (‘0’ desabilita encapsulamento):
```
Comment (renie) [inline pages.pt_BR/freebsd/base64.md:14]:
```suggestion
- Quebra (insere uma quebra de linha) a saída codificada em uma largura específica (‘0’ desabilita encapsulamento):
```
I know this is not the literal translation, but it makes it more understandable than "enrola".

File: pages.pt_BR/freebsd/cal.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Exibe um calendário para um ano e mês específicos:
+
+`cal {{mês}} {{ano}}`
+
+- Exibe o calendário inteiro para o ano atual:
+
+`cal -y`
+
+- Não [d]estaca hoje e exibe [3] meses abrangendo a data:
```
Comment (renie) [inline pages.pt_BR/freebsd/cal.md:22]:
```suggestion
- Não destaca hoje e exibe [3] meses abrangendo a data:
```
No need to highlight if the letter does not say much in portuguese.

File: pages.pt_BR/freebsd/sockstat.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# sockstat
+
+> Lista sockets de domínio aberto Internet ou UNIX.
+> Mais informações: <https://man.freebsd.org/cgi/man.cgi?sockstat>.
+
+- Vê quais usuários/processos estão [e]scutando em quais portas:
+
+`sockstat -l`
+
+- Exibe informação para sockets IPv[4] e IPv[6] [e]scutando em portas específicas usando um [p]rotocolo específico:
```
Comment (renie) [inline pages.pt_BR/freebsd/sockstat.md:10]:
```suggestion
- Exibe informação para sockets IPv[4] e IPv[6] escutando em portas específicas usando um [p]rotocolo específico:
```

File: pages.pt_BR/freebsd/sockstat.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`sockstat -{{4|6}} -l -P {{tcp|udp|sctp|divert}} -p {{port1,port2...}}`
+
+- Exibe também sockets [c]onectados, não resolvendo UIDs [n]uméricos para nome de usuários e usando um campo mais [l]argo:
```
Comment (renie) [inline pages.pt_BR/freebsd/sockstat.md:14]:
```suggestion
- Exibe também sockets [c]onectados, não resolvendo UIDs [n]uméricos para nome de usuários e usando um campo mais largo:
```

File: pages.pt_BR/freebsd/sockstat.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`sockstat -cnw`
+
+- Exibe somente sockets que pertencem a um ID [j]ail específico ou nome de modo detalhado:
+
+`sockstat -jv`
+
+- Exibe o [e]stado do protocolo e o número da porta do encapsulamento [U]DP remoto, se aplicável (atualmente, estes estão implementados somente para SCTP e TCP):
```
Comment (renie) [inline pages.pt_BR/freebsd/sockstat.md:22]:
```suggestion
- Exibe o estado do protocolo e o número da porta do encapsulamento [U]DP remoto, se aplicável (atualmente, estes estão implementados somente para SCTP e TCP):
```

File: pages.pt_BR/freebsd/sockstat.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`sockstat -sU`
+
+- Exibe o módulo de controle de [c]ongestionamento e a [p]ilha de protocolo, se aplicável (atualmente, estes estão implementados somente para TCP):
```
Comment (renie) [inline pages.pt_BR/freebsd/sockstat.md:26]:
```suggestion
- Exibe o módulo de controle de [c]ongestionamento e a pilha de protocolo, se aplicável (atualmente, estes estão implementados somente para TCP):
```

File: pages.pt_BR/freebsd/sockstat.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+`sockstat -CS`
+
+- Exibe apenas sockets da Internet se os endereços local e estrangeiro não estiverem no prefixo de rede loopback 127.0.0.0/8, ou não contiverem o endereço de loopback IPv6 ::1:
+
+`sockstat -L`
+
+- Não exibe o cabeçalho (modo [s]ilencioso), mostrando sockets [u]nix e exibindo o `inp_gencnt`:
```
Comment (renie) [inline pages.pt_BR/freebsd/sockstat.md:34]:
```suggestion
- Não exibe o cabeçalho (modo silencioso), mostrando sockets [u]nix e exibindo o `inp_gencnt`:
```

**Review by renie — CHANGES_REQUESTED** [review]
Just some small fixes before accepting.

**Review by renie — APPROVED** [review]
Thanks for the contribution, @rafaelgpaves !

**Pull request merged**